package.json

```diff
@@ -13,6 +13,7 @@
     "./server": "./dist/server.js",
     "./client": "./dist/client.js",
     "./react": "./dist/react.js",
+    "./socketio": "./dist/socketio.js",
```

Comment:
I think we need to update the examples in the `README` now that we've factored out the socketio stuff into its own import.